src/reducer.js

```diff
@@ -210,6 +244,17 @@ export function resourceReducer(state, action) {
       );
     }
 
+    case UPDATE_COLLECTION_ERROR: {
+      const { id, data, opts } = action;
+      const collectionIdHash = getCollectionIdHash(id);
+      const queryHash = getQueryHash(opts);
+      return state.withMutations((resourceState) => resourceState
+        .deleteIn(['updating', collectionIdHash, queryHash])
+        .update('updating', (map) => (map.get(collectionIdHash, iMap()).isEmpty() ? map.delete(collectionIdHash) : map))
+        .setIn(['collections', collectionIdHash, queryHash, 'error'], data)
```

Comment:
This is setting the same state as `LOAD_COLLECTION_ERROR` and would cause requests to the collection with `queryCollection` to report as error. 

Can we assume no error set is required in state, similar to `UPDATE_ERROR`?

Reply:
I think this could be okay. Would the `deleteIn(['updating', collectionIdHash])` be the trigger to allow a GET collection fetch?